src/components/list-orders.js

```diff
@@ -24,6 +25,7 @@ class ListOrders extends Component {
             </li>)
           }
         </ul>
+        {!isFetching && <CalendarOrder />}
```

Comment:
Do render the calendar always. The loading will be an overlay if isFetching is true.